packages/app/src/systems/Account/services/account.ts

```diff
@@ -231,9 +231,9 @@ export class AccountService {
   }
 
   static filterByName(accounts: Account[], name = '') {
-    return accounts.filter((account) =>
-      account.name.toLowerCase().includes(name.toLowerCase())
-    );
+    return accounts.filter((account) => {
+      account.name === name;
```

Comment:
it's missing a return clause here

```suggestion
      return account.name === name;
```